src/pydolphinscheduler/tasks/http.py

```diff
@@ -82,8 +106,27 @@ def __init__(
             raise PyDSParamException(
                 "Parameter http_method %s not support.", http_method
             )
+        
+        if isinstance(http_params, list):
+            warnings.warn(
+                "The `http_params` parameter currently accepts a dictionary instead of a list. Your parameter is being ignored.",
+                DeprecationWarning,
+            )
+            
         self.http_method = http_method
-        self.http_params = http_params or []
+        self.http_params = http_params  # Storing the original value
```

Comment:
directly use param name `_http_params` is better, because we can remove the `@http_params.setter`
```suggestion
        self._http_params = http_params 
```

Reply:
@zhongjiajie sir done